### PR TITLE
product update for 20.01.2023

### PR DESCRIPTION
**Pull request by MartaD**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs.
```
Comment (sujaya-sys) [inline docs/change-log.md:7]: I'd stay in line with the docs here and refer to "Artefact API"

My suggestion would be something like: "The [Artefact Version API](https://api-docs.humanitec.com/#tag/ArtefactVersion) now supports PR refs. Next to branch- and tag refs artefact versions build on a PR branch can be pushed by users"

File: docs/change-log.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` value after submiting the same name.
```
Comment (sujaya-sys) [inline docs/change-log.md:11]: Would "The Service account name on the Workload Details Screen is no longer reset to `default` when re-submitting the currently defined name." make sense?

File: docs/change-log.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` value after submiting the same name. 
+
+- **Improved:** Humanitec application's color palette has been update to better match the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) criteria.
```
Comment (sujaya-sys) [inline docs/change-log.md:13]: The color palette of the Humanitec UI has been updated to better match the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/)

File: docs/change-log.md (hunk 1 above)
Comment (MartaD) [inline docs/change-log.md:7]:
```suggestion
- **New:** The [Artefact Version API](https://api-docs.humanitec.com/#tag/ArtefactVersion) now supports PR refs. Next to branch- and tag refs artefact versions built on a PR branch can be pushed by users. 
```

File: docs/change-log.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** The [Artefact Version API](https://api-docs.humanitec.com/#tag/ArtefactVersion) now supports PR refs. Next to branch- and tag refs artefact versions built on a PR branch can be pushed by users. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` value after submiting the same name.
```
Comment (MartaD) [inline docs/change-log.md:11]:
```suggestion
- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` after re-submitting the currently defined name. 
```

File: docs/change-log.md [hunk 5]
```diff
@@ -2,6 +2,16 @@
...
 
+## Product Update January 20th, 2023
+
+- **New:** The [Artefact Version API](https://api-docs.humanitec.com/#tag/ArtefactVersion) now supports PR refs. Next to branch- and tag refs artefact versions built on a PR branch can be pushed by users. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` after re-submitting the currently defined name. 
+
+- **Improved:** Humanitec application's color palette has been update to better match the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) criteria.
```
Comment (MartaD) [inline docs/change-log.md:13]:
```suggestion
- **Improved:** The color palette of the Humanitec UI has been updated to match better the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).
```

**Review by sujaya-sys — APPROVED** [review]

**Pull request merged**